src/languages/en.js

```diff
@@ -107,6 +108,8 @@ export default {
         blockedFromConcierge: 'Communication is barred',
         youAppearToBeOffline: 'You appear to be offline.',
         fileUploadFailed: 'Upload Failed. File is not supported.',
+        localTime: ({user, time}) => `It's ${time} for ${user}`,
```

Comment:
I think you're not using this right? I think this was changed in main since you made this branch.

It looks like it was removed here: https://github.com/Expensify/Expensify.cash/pull/4020/files